### ISSUE TITLE
CATCH node doesn't intercept "Error: connect ECONNREFUSED 127.0.0.1:6379"

### ISSUE THREAD

**Issue by bogorad** [chameleonbr/node-red-contrib-redis#52]
More generally - is there a way to get notified on connection to Redis and disconnection from Redis?

For now, I'm sending PING command to Redis, with a timeout of 250ms, and checking if the status has changed. But a native solution would be great. Also, the error (subj) is distracting.

![image](https://user-images.githubusercontent.com/2777398/138945936-3ffaafa3-76c6-4f29-8b9e-5536cc6d4096.png)

**Comment by laurafernandezdiaz** [comment]
Yes, in my case it is not always connected to Redis so a native way of checking if it is connected or not would be very useful.

**Comment by eviltik** [comment]
+1

**Comment by duchang9264** [comment]
> 连接

+1 ，而且我发现没有配置密码的地方，我怀疑是因为没有输密码，所以报这个错误，因为我的redis配置了密码
